maven-embedder/src/main/java/org/apache/maven/cli/transfer/ConsoleMavenTransferListener.java

```diff
@@ -35,7 +35,7 @@
  */
 public class ConsoleMavenTransferListener extends AbstractMavenTransferListener {
```

Comment:
Can we add a comment specifying that this class is not thread safe and should only be used from a single thread, or wrapped in the `SimplexTransferListener` ?

Reply:
Done in https://github.com/apache/maven/pull/1575/commits/274f1807ab0fcef625efadb26aabcaa2be1fe31b